mmv1/third_party/terraform/resources/resource_dataflow_flex_template_job.go.erb

```diff
@@ -112,14 +228,17 @@ func resourceDataflowFlexTemplateJobCreate(d *schema.ResourceData, meta interfac
 		return err
 	}
 
+	env, err := resourceDataflowFlexJobSetupEnv(d, config)
+	if err != nil {
+		return err
+	}
+
 	request := dataflow.LaunchFlexTemplateRequest{
 		LaunchParameter: &dataflow.LaunchFlexTemplateParameter{
 			ContainerSpecGcsPath: d.Get("container_spec_gcs_path").(string),
 			JobName:              d.Get("name").(string),
 			Parameters:           expandStringMap(d, "parameters"),
-			Environment: &dataflow.FlexTemplateRuntimeEnvironment{
-				AdditionalUserLabels: expandStringMap(d, "labels"),
-			},
+			Environment:          &env,
```

Comment:
Might have missed it, but is `transform_name_mapping` not included in the create request? Is there a reason for that?

Reply:
It is only used for updating a pipeline if there is a change in the stage/transform names.